js_functions/task2.js

```diff
@@ -0,0 +1,11 @@
+function check(age) {
```

Comment:
what is `check`? It does not represent what function does. Use more informative name, like `getIsAdult`